packages/webpack-plugin/src/stylable-webpack-plugin.ts

```diff
@@ -177,6 +151,32 @@ export class StylableWebpackPlugin {
         });
         this.injectStylableCSSOptimizer(compiler);
     }
+    private processStylableModuleAssets(
+        compilation: webpack.compilation.Compilation,
+        module: StylableModule
+    ) {
+        const rootContext = (compilation as any).options.context;
```

Comment:
`compilation.context`

Reply:
I think I saw a difference but will change and see what is going on.